.gitignore

```diff
@@ -5,7 +5,7 @@ __pycache__/
 
 # C extensions
 *.so
-
```

Comment:
Why exclude this file? In xorbits repo, this file is needed.

Reply:
for the same reason I will return it back.